src/snapshots/new_nu_parser__test__node_output@binary_ops_mismatch.nu.snap

```diff
@@ -32,7 +33,7 @@ input_file: tests/binary_ops_mismatch.nu
 9: forbidden
 10: string
 11: unknown
-12: ()
+12: unknown
```

Comment:
Maybe `any` type would be better in case of error. The intent is that after compiling, there wouldn't be any `unknown` types anymore. `unknown` type means "untouched by a typechecker". That's the reason the `forbidden` type exists: To mark nodes visited by the typechecker that shouldn't be directly evaluated, like operators. Not sure if we need to stick with this that strictly, but that's how I've been thinking about it.

Reply:
Another option would be to introduce an `error` type.

Reply:
I think it's better to not have multiple `Any` interpretations. Let's introduce an error type. 

My intuition for leaving it untouched was that the compiler is going to throw an error anyways but I do see how that info might be useful.

Reply:
Yeah, the `error` type is better, and it would match the `garbage` AST node nicely.